### PR TITLE
Add blog post for installing and using NPKILL

### PR DESCRIPTION
**Pull request by leomet07**

### CODE REVIEW

File: _posts/using_npkill.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Using NPKILL
+
+## What is NPKILL?
+
+npkill is a command line tool that searches for all instances of those clunky and bloated _node_modules/_ folders and then offers you a convientient TUI (terminal user interface) to select and delete them.
+
+## Running NPKILL
+
+NPKILL doesn't have to be installed, as you can just use NPX to run it.
```
Comment (TheEgghead27) [inline _posts/using_npkill.md]: This statement sounds misleading - doesnt NPX download and install it for NPX usage?

File: _posts/using_npkill.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Using NPKILL
+
+## What is NPKILL?
+
+npkill is a command line tool that searches for all instances of those clunky and bloated _node_modules/_ folders and then offers you a convientient TUI (terminal user interface) to select and delete them.
```
Comment (TheEgghead27) [inline _posts/using_npkill.md]: Would `codeblock` (\`text\`) fit better than italics?

**Review by TheEgghead27 — CHANGES_REQUESTED** [review]
Looks great - just have a few things that might be good to clarify

File: _posts/using_npkill.md (hunk 1 above)
Comment (leomet07) [reply _posts/using_npkill.md]: addressed

File: _posts/using_npkill.md (hunk 2 above)
Comment (leomet07) [reply _posts/using_npkill.md]: addressed

**Review by TheEgghead27 — APPROVED** [review]

**Pull request merged**